test/metabase/api/search_test.clj

```diff
@@ -81,10 +80,17 @@
    (t2/select-one [:model/Table [:name :table_name] [:schema :table_schema] [:description :table_description]]
                   :id (mt/id :checkins))))
 
-(defn- sorted-results [results]
-  (->> results
-       (sort-by (juxt (comp (var-get #'scoring/model->sort-position) :model)))
-       reverse))
+(defn- clean-result [result]
+  (dissoc (u/remove-nils result) :bookmark :database_id :model_id :model_index_id :table_id :dataset_query :last_editor_id :last_edited_at
+          :creator_common_name :creator_id
+          ;; false for new search segments... not sure why
+          :created_at))
+
+(defn- cleaned-results [results]
+  ;; WIP need to get these fields matching
```

Comment:
Not sure what I meant by this comment. Perhaps just a reference to the fields we're dissoc-ing.